src/fragments/lib/in-app-messaging/js/create-campaigns.mdx

```diff
@@ -0,0 +1,53 @@
+A campaign is a messaging initiative that engages a specific audience [segment](https://docs.aws.amazon.com/pinpoint/latest/userguide/segments.html). A campaign sends tailored messages according to a schedule that you define. You can use the AWS console to create a campaign that sends messages through any single channel that is supported by Amazon Pinpoint: mobile push, In-App, email, SMS or custom channels. [Learn how to create a campaign using Amazon Pinpoint](https://docs.aws.amazon.com/pinpoint/latest/userguide/campaigns.html) to continue integrating in-app messages in your app with Amplify.
+
+1. Login to the [AWS Console](https://console.aws.amazon.com/console/home), and Search for “Amazon Pinpoint”.
+2. Click on your project from the list of available project. Your project name would have been setup with the CLI as your pinpoint resource name when the `amplify add analytics` command was used.
+3. Click on “Campaigns” from the left navigation menu, and then click on “Create a campaign”
+
+   ![In-App-Messaging](/images/lib/in-app-messaging/create-pinpoint-campaign.png)
+
+4. Add a name to your campaign, and keep the following options as follows and then click Next:
+
+   1. Campaign type: **Standard campaign**
+   2. Channel: **In-App messaging**
+   3. set prioritization: **Fairly important**
+
+5. Click on the “Create a segment” radio button, and add a name for your segment, and then click on Next.
+
+   1. You can add as many segments as needed to the campaign. For this quickstart, you can use “Include any audiences” under the “Segment group 1” section.
+   2. You can add a criteria to your segments to ensure that audiences that satisfy that criteria can receive the in-app message.
+   3. If you see an error message titled “Segment might include multiple channels”, click on “I understand” to proceed.
+
+   ![In-App-Messaging](/images/lib/in-app-messaging/choose-campaign-segment.png)
+
+6. Click on the “Create a new in-app message” radio button.
+7. You have the ability to customize the following attributes of the in-app message:
+   - **Layout**: Which includes all of the different messaging layout options.
+   - **Header**: Title of the in-app message, including the text color/alignment.
+   - **Message**: The body of the Message, including the text color/alignment.
+   - **Background**: Control the background color of the in-app message.
+   - **Image URL**: Add an image to be displayed as part of the in-app message body.
+     - As React Native does not support SVG rendering out of the box, Amplify cannot render SVG images by default. For SVG image support with In-App Messaging a custom UI implementation is required.
```

Comment:
Is there a something like a `platform` tag we can use to scope this line to React Native documentation only?

Reply:
There is no 'React Native' platform option under Libraries. In-App Messaging is going to be listed under Javascript platform just like Push Notifications.

Reply:
In that case this should read more like a callout and not a list item IMO

Reply:
Can you please add a callout tag @chintannp

Reply:
Added the callout @calebpollman .